src/targets/services/attributesHelpers.js

```diff
@@ -0,0 +1,32 @@
+import log from 'cozy-logger'
+import omit from 'lodash/omit'
+
+export const mergeAttributes = async (
+  newIdentity,
+  currentMyselfContact,
+  contactCollection
+) => {
+  log('info', `Found a me contact`)
+  const newAttributes = findNewAttributes(
+    newIdentity.contact,
+    currentMyselfContact
+  )
+  if (Object.keys(newAttributes).length) {
+    log(
+      'info',
+      `And found the following new attributes : ${Object.keys(
+        newAttributes
+      ).join(', ')}`
+    )
+    Object.assign(currentMyselfContact, newAttributes)
+    log('info', `Updating the me contact with new attributes`)
+    await contactCollection.update(currentMyselfContact)
```

Comment:
I think this function should only do the merge and not the update via the collection. It would simplify the signature, make the function sync and thus a bit more testable, and this way it is the responsibility of the caller to deal with the collection.